_inc/client/at-a-glance/stats/index.jsx

```diff
@@ -58,6 +63,15 @@ export class DashStats extends Component {
 
 		let totalViews = 0;
 
+		/* translators: short date format, such as: Jan 12. */
+		const shortMonthFormat = __( 'M j', 'jetpack' );
+
+		/* translators: long date format, such as: January 12th. */
+		const longMonthFormat = __( 'F jS', 'jetpack' );
+
+		/* translators: long month/year format, such as: January, 2021. */
+		const longMonthYearFormat = __( 'F, Y', 'jetpack' );
```

Comment:
A comma isn't needed between the month and year in English (see [this](https://www.grammarly.com/blog/commas-in-dates/#:~:text=When%20writing%20a%20date%2C%20a,don't%20need%20a%20comma.) for more info). So, I think we can remove this comma:

```suggestion
		const longMonthYearFormat = __( 'F Y', 'jetpack' );
```

If other languages require the comma, the translations can include it, right?

Reply:
Thank you! I've committed your suggestion, this should be ready for another review :)